src/main/webapp/scripts/user-script.js

```diff
@@ -101,9 +166,10 @@ function addUsertoOUByPath(node, path, user){
     }
 }
 
-// change value of each OU to the number of users in the OU
+// change value of each OU to the number of users in the OU, 
+// value is 1 more than the number of users to prevent empty orgUnits display issue 
 function incrementUserCount (node){
-    var value = node.data['users'].length;
+    var value = node.data['users'].length + 1;
```

Comment:
what is value in this context? can the variable be renammed to something more descriptive